web/src/TaskApp.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { css } from "@emotion/core";
+import styled from "@emotion/styled";
+import { useRealmApp } from "./RealmApp";
+import ProjectScreen from "./components/ProjectScreen";
+import Sidebar from "./components/Sidebar";
+
+export default function TaskApp() {
+  const { app } = useRealmApp();
+  const [currentProject, setCurrentProject] = React.useState(
+    app.currentUser.customData.memberOf[0]
```

Comment:
Might be useful to have a comment here: 
```suggestion
    // set the current project as  "My Project"
    app.currentUser.customData.memberOf[0]
```